test_data/valid_files/organizations.json

```diff
@@ -0,0 +1 @@
+legacy file
```

Comment:
what's the use for this file in the current state?

Reply:
It is added to the tarball (valid_files.tar.gz) to make sure the unit test is testing with a tar file which is as similar as possible to the one Github produces. If this file was there before, the existing unit test would've spotted the bug.